action/protocol/context.go

```diff
@@ -122,6 +125,7 @@ type (
 		SuicideTxLogMismatchPanic               bool
 		PanicUnrecoverableError                 bool
 		CandidateIdentifiedByOwner              bool
+		ExtendCandidateDelegatesTo48            bool
```

Comment:
delete

Reply:
see above reply, after hard-fork we can try to remove this flag